docker/README.md

```diff
@@ -10,5 +10,6 @@ Pushing images is currently handled via `docker push`, and is not part of the bu
 mvn package -DskipTests  # Build local images
 
 # Build images for a private registry; trailing '/' is required:
-# mvn package -DskipTests -Ddocker.registry=docker.example.com:8080/ -Ddocker.tag=$VERSION-$BUILD_NUMBER
+# mvn package -DskipTests -Ddocker.registry=docker.example.com:8088/ -Ddocker
```

Comment:
I think 8080 makes more sense here? I guess it doesn't really matter if its just an example

Reply:
Yes, It can be either. I can change it to 8080 :)

Reply:
@rodesai did you see @apurvam 's comment below about system tests?